src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.use_entity_culling.name": "Отбраковка сущностей",
+  "sodium.options.use_entity_culling.tooltip": "Если включено, сущности, обнаруженные вне видимых чанков, не будут выводиться на рендеринг. Это может помочь улучшить производительность, исключив рендеринг сущностей, находящихся под землёй или за стенами.",
+  "sodium.options.use_particle_culling.name": "Фильтрация частиц",
+  "sodium.options.use_particle_culling.tooltip": "Если включено, будут отображаться только те частицы, которые определены как видимые. Это может обеспечить значительное улучшение частоты кадров, когда рядом находится много частиц.",
+  "sodium.options.animate_only_visible_textures.name": "Анимировать только видимое",
+  "sodium.options.animate_only_visible_textures.tooltip": "Если включено, будут обновляться лишь видимые анимированные текстуры. Это может значительно повысить частоту кадров на некоторых устройствах, особенно с \"тяжёлыми\" пакетами ресурсов. Если у каких-то текстур возникают проблемы с анимацией, попробуйте отключить эту опцию.",
+  "sodium.options.cpu_render_ahead_limit.name": "Лимит пре-рендеринга",
+  "sodium.options.cpu_render_ahead_limit.tooltip": "Задаёт максимальное количество кадров, которые ЦП может подготавливать для рендеринга на ГП. Очень низкие или высокие значения могут дестабилизировать частоту кадров.",
+  "sodium.options.cpu_render_ahead_limit.value": "%s кадр(ов)",
+  "sodium.options.allow_direct_memory_access.name": "Разрешить прямой доступ к памяти",
+  "sodium.options.allow_direct_memory_access.tooltip": "Если включено, некоторым критическим путям кода будет разрешено использовать прямой доступ к памяти для повышения производительности. Зачастую это сильно снижает перегрузку ЦП при рендеринге чанков и сущностей, но может усложнить диагностику некоторых ошибок и вылетов. Отключать это стоит только если вы следуете инструкциям или точно знаете, что делаете.",
+  "sodium.options.enable_memory_tracing.name": "Включить трассировку памяти",
+  "sodium.options.enable_memory_tracing.tooltip": "Функция отладки. Если включено, трассировка стека будет собираться вместе с распределением памяти, чтобы помочь улучшить диагностическую информацию при обнаружении утечек памяти.",
+  "sodium.options.performance_impact_string": "Влияние на производительность: %s",
+  "sodium.options.use_persistent_mapping.name": "Использовать постоянный буфер",
+  "sodium.options.use_persistent_mapping.tooltip": "Если включено, небольшое количество памяти будет постоянно использоваться в качестве промежуточного буфера для загрузки чанков, помогая уменьшить перегрузку ЦП и нестабильность времени отрисовки кадра при загрузке/выгрузке чанков.\n\nТребуется OpenGL 4.4 или ARB_buffer_storage.",
```

More correct and well-sounding option
```suggestion
  "sodium.options.use_persistent_mapping.tooltip": "Если включено, небольшое количество памяти будет постоянно использоваться в качестве промежуточного буфера для загрузки чанков, помогая уменьшить накладные расходы на ЦП и нестабильность времени отрисовки кадра при их загрузке/выгрузке.\n\nТребуется OpenGL 4.4 или ARB_buffer_storage.",
```